src/components/layout/index.jsx

```diff
@@ -19,82 +16,22 @@ const MenuContainer = styled.div`
 
 const ContentContainer = styled.div`
     grid-area: 2 / 1 / 3 / 2;
-
-    > * {
-        padding: 0px 16px;
-    }
 `;
 
 const FooterContainer = styled.div`
     grid-area: 3 / 1 / 4 / 2;
 `;
 
-//TODO temporary component
-const BlurredWrapper = styled.div`
-    filter: blur(2px);
-`;
-
-const TemporaryTitle = styled.h1`
-    color: var(--white);
-    font-size: 56px;
-    padding: 0 16px;
-    text-align: center;
-`;
-
-const Overlay = styled.div`
-    position: fixed;
-    width: 100%;
-    height: 100%;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    background-color: rgba(0, 0, 0, 0.8);
-    z-index: 2;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-`;
-
-const WhiteLogo = styled.img`
-    max-width: 300px;
-`;
-
-const Banner = styled.img`
-    margin-top: 16px;
-    width: 100%;
-`;
-
-const VideoContainer = styled.div`
-    width: 50%;
-    height: 50%;
-
-    @media (max-width: 1024px) {
-        height: 30%;
-    }
-
-    @media (max-width: 700px) {
-        width: 80%;
-    }
-`;
-//---------------------
-
-const Layout = ({ children }) => (
+export const Layout = ({ children }) => (
```

Comment:
Come mai stai cambiando gli export?

Non che non mi tornino o siano sbagliati. Ma mi sfugge il vantaggio e quindi la modifica (e mi aumenta lo sforzo cognitivo a fare review della PR).

Reply:
Preferisco i named export in quanto a lungo termine potrebbero creare entropia (ad esempio se cambia il nome del componente, con i default probabilmente te lo fumi nei vari import, mentre col named export no, dato che fallisce la build).